File: mobile/lib/data/model/requests/onboarding_request.dart
import 'dart:typed_data';

class RegisterRequest {
  String? firstName;
  String? lastName;
  String? username;
  String? password;
  ByteBuffer? profilePicture;
  String? phoneNumber;
  String? dob;
  String? referralCode;
  String? address;
  Float64x2? longitude;
  Float64x2? latitude;
}

class LoginRequest {
  String? email;
  String? password;
}

File: web/src/api/requests/OnboardingRequest.ts
import {Property} from "csstype";
import Float = Property.Float;

export interface LoginRequest {
    email : String
    password : String
}

export interface RegisterRequest {
    first_name : String
    last_name : String
    username : String
    email :  String
    password : String
    profile_picture : File
    phone_number : String
    dob : String
    referral_code : String
    address : String
    longitude : Float
    latitude : Float
}